BibBuddy/app/src/main/java/de/bibbuddy/BookDao.java

```diff
@@ -485,4 +485,63 @@ public Long findShelfIdByBook(Long id) {
     return shelfId;
   }
 
+  /**
+   * Method that finds the shelf name of a book in the database.
+   *
+   * @param id id of the book
+   * @return the shelf name of the book
```

Comment:
```suggestion
   * @return the shelf name of the book.
```
Comment lines should end with a full stop.